mobify/platform-scaffold
# Update sdk version to 0.15.0

## Changes
- [x] Update sdk version to 0.15.0
- [x] Analytic Connector setup #556 
- [x] @jeffkamo changes #630

## How to test-drive this PR
- Run `npm rm -r node_modules/progressive-web-sdk && npm i`
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)

Expect Result: Nothing breaks. All test passes
